package-lock.json

```diff
@@ -105,7 +105,7 @@
         "@angular/platform-browser": "^15.1.0",
         "@angular/platform-browser-dynamic": "^15.1.0",
         "@angular/router": "^15.1.0",
-        "@morgan-stanley/composeui-messaging-client": "*",
+        "@finos/fdc3": "2.0",
```

Comment:
That's interesting. The npm package has patch versions, like you use it on the chart.
https://www.npmjs.com/package/@finos/fdc3
I don't know how it hasn't failed :D 
But just to be sure please include the patch version for 2.0.3

Reply:
Sure will replace it with the patched version!